scripts/loadtest-scripts/prepare_loadtests.py

```diff
@@ -3,12 +3,15 @@
 import os
 import time
 
+from sdc import csvfile
 from sdc.clients import SDCClient, collectionexerciseclient
 from sdc.clients.collectionexerciseclient import collection_exercise_url
 from sdc.clients.collectioninstrumentclient import CollectionInstrumentClient
 from sdc.clients.iac_client import RemoteFileNotFoundException
 from sdc.clients.sampleclient import SampleClient
 
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
```

Comment:
typo?

Reply:
I lied, it was this https://github.com/sindresorhus/refined-github/issues/1452

Reply:
Ha - any chance of an approval then? ;-)